### ISSUE TITLE
The link located in ”https://nightly.link/futurerestore/futurerestore/workflows/ci/main“ is no longer working

### ISSUE THREAD

**Issue by P5A-i** [futurerestore/futurerestore#171]
The link located in ”https://nightly.link/futurerestore/futurerestore/workflows/ci/main“ is no longer working

**Comment by winaviation** [comment]
yeah, github apparently deprecated the v2 artifacts which futurerestore was using, this also makes futurerestore unavailable (cant fetch versioning file), you can mention the dev to fix this issue, he might either update to v3 or v4 artifacts

**Comment by Cryptiiiic** [comment]
@winaviation @P5A-i I know its just everything is being done on dev branch with manual actions builds and nightly is dumb and doesn't add manual builds
